tests/src/test/scala/common/Pair.java

```diff
@@ -15,16 +15,6 @@
  * limitations under the License.
  */
 
-/*******************************************************************************
```

Comment:
this copy right needs to stay.

Reply:
or we can throw this away and the basic launcher with it.

Reply:
Do we still have Java code around 🤔